extensions/wikia/CommunityHeader/tests/NavigationTest.php

```diff
@@ -22,15 +14,11 @@ private function manualSetUp() {
 	 * @param $expectedExploreItems
 	 */
 	public function testExploreItems( $globals, $expectedExploreItems ) {
-		$globals = new GlobalStateWrapper( [
-			'wgEnableCommunityPageExt' => $globals['wgEnableCommunityPageExt'],
-			'wgEnableDiscussions' => $globals['wgEnableDiscussions'],
-			'wgEnableForumExt' => $globals['wgEnableForumExt'],
-		] );
+		$this->mockStaticMethodWithCallBack( 'WikiFactory', 'getVarValueByName', function( $variable, $id) use ($globals) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$id". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)